types.go

```diff
@@ -582,12 +582,21 @@ type Release struct {
 	Description        string
 	Status             string
 	DeploymentStrategy string
+	Metadata           ReleaseMetadata
 	User               User
 	EvaluationID       string
 	CreatedAt          time.Time
 	ImageRef           string
 }
 
+type PostDeploymentInfo struct {
+	FlyctlVersion string `json:"flyctl_version"`
+	Error         string `json:"error"`
```

Comment:
Perhaps we should have an `omitempty` on this field since it will be empty for many cases. So we don't have a `null` as a value